tests/conftest.py

```diff
@@ -958,12 +969,11 @@ def milvus_server_uri():
     from milvus import default_server
 
     logger = logging.getLogger(f"morpheus.{__name__}")
-    try:
-        # Milvus checks for already bound ports but it doesnt seem to work for webservice_port. Use a random one
-        default_server.webservice_port = _get_random_port()
 
-        # Now start the server
-        default_server.start()
```

Comment:
What happened to starting the server?

Reply:
Still happens but we are using `default_server` as a context manager.